tests_v2/cmux.py

```
@@ -54,29 +74,77 @@ def _read_last_socket_path() -> Optional[str]:
+            return True
+        except OSError:
+            time.sleep(0.05)
+        finally:
+            try:
+                s.close()
+            except Exception:
+                pass
+    return False
+
+
 def _default_socket_path() -> str:
```

Ruff (0.15.12)

[warning] Too many return statements (9 > 6)

(PLR0911)